rose/server/player_test.py

```diff
@@ -1,15 +1,74 @@
-from . import player
+from rose.common import actions, config
+from rose.server.player import Player
 
 
-def test_in_lane():
-    p = player.Player("A", car=0, lane=0)
-    for x in (0, 1, 2):
-        p.x = x
-        assert p.in_lane()
+def test_player_initialization():
+    player = Player("John", "car1", 1)
```

Comment:
This is not documented, but based on rose/server/game.py, car is an integer.

Reply:
oyy, yes, it's an int ...

a. changed to int
b. added documentation, so it will be clear it's an int